### PR TITLE
Checks path to archive before processing

### PR DESCRIPTION
**Pull request by cdelcol**
Fixes blackberry/BB10-Webworks-Packager#205

Checks for the existence of the archive, whether it is the .zip or the directory.

### CODE REVIEW

File: lib/localize.js [hunk 1]
```diff
@@ -168,6 +168,9 @@ var Localize = require("localize"),
         },
         "WARN_WEBWORKS_JS_PACKAGED" : {
             "en": "webworks.js is now packaged as local:///chrome/webworks.js"
+        },
+        "EXCEPTION_INVALID_ARCHIVE_PATH" : {
+            "en": "An archive does not exist at the path specified: \"$[1]\""
```
Comment (nukulb) [inline lib/localize.js]: its no longer necessary its an archive, it could be a folder as well

Comment (cdelcol) [inline lib/localize.js]: "An archive or application folder..."? I tried to shorten it a bit on purpose.

**Comment by cdelcol** [comment]
Finally. All unit tests pass. Tested:
1. Mac and Win XP
2. with/without spaces
3. with/without " surrounding paths
4. with/without specifying -o parameter. If you have a space, the full directory is obviously not honoured, but no ugly failures.

**Comment by nukulb** [comment]
you need a rebase again

**Comment by cdelcol** [comment]
Rebased and ready to go whenever. Not a huge priority.

**Comment by ejzn** [comment]
merged manually.

**Comment by nukulb** [comment]
this was not tested

**Comment by ejzn** [comment]
Hmm, fun. I will test it right now.

**Comment by ejzn** [comment]
tested on CI branch and works as desired next-cdelcol-invalid-path-check. Error message is properly displayed for missing zip or directory.

**Pull request closed without merging**